BuildAssets/repos/publish-apt.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Required env vars:
+#export AWS_ACCESS_KEY_ID=$(get_octopusvariable "OctopusToolsAwsAccount.AccessKey")
```

Comment:
future improvement: we can make this fail if the env var isn't there

Reply:
Addressed in #55.